broker/src/main/java/io/camunda/zeebe/broker/jobstream/StreamPusher.java

```diff
@@ -47,14 +51,23 @@ private void push(final P payload, final ErrorHandler<P> errorHandler) {
       transport
           .send(request, streamId.receiver())
           .whenCompleteAsync((ok, error) -> onPush(payload, errorHandler, error), executor);
+      if (LOG.isTraceEnabled()) {
+        LOG.trace("Pushed {} to stream {}", payload, streamId);
+      }
     } catch (final Exception e) {
+      metrics.jobPushFailed();
+      LOG.debug("Failed to push {} to stream {}", payload, streamId, e);
```

Comment:
This is usually expected during gateway restarts or client disconnections. It will be noisy to make it a warning. I'm assuming engine error handling will handle it. So a warning would be unnecessary.